electron-app/src/constants/dirpath.ts

```diff
@@ -9,15 +9,21 @@ import {
 } from '../utils';
 import { path7za } from '7zip-bin';
 
+export const DEFI_CONF_DIR = '.defi';
+export const DEFI_CONF = 'defi.conf';
 export const APP_NAME = app.name;
 export const IS_DEV = process.env.NODE_ENV === 'development';
 export const IS_PACKAGED =
   process.mainModule.filename.indexOf('app.asar') !== -1;
 
-export const HOME_PATH = app.getPath('home');
-export const APP_DIR = path.join(HOME_PATH, './.defi');
-export const CONFIG_FILE_NAME = path.join(APP_DIR, '/defi.conf');
-export const UI_CONFIG_FILE_NAME = path.join(APP_DIR, '/defi.ui.yaml');
+export const BASE_HOME_PATH = app.getPath('home');
+export const HOME_PATH =
+  getPlatform() === 'mac'
+    ? getDefaultDebugLogFilePath(BASE_HOME_PATH)
+    : BASE_HOME_PATH;
+export const APP_DIR = path.join(HOME_PATH, `./${DEFI_CONF_DIR}`);
+export const APP_DIR_BLOCKCHAIN = path.join(HOME_PATH, `./${DEFI_CONF_DIR}`);
```

Comment:
Can we use one cont variable instead of two for same path?

Reply:
oh good find, I forgot to remove

Reply:
great